### PR TITLE
Feature/style doctors reservation form

### LINKED ISSUE #18: [4 pts] Main Page, Reservation Form - Member 4
Style the named components according to the design. Also, create any component or integration tests required.

### PR DESCRIPTION
**Pull request by Camilovelag**
PR to partially close #18   💯 

### TBD ☑️ 
- Add tests.

### _Notes_
- Please, feel free to add whichever you consider needs to be improved.

I'll be waiting for your feedback. Happy coding 👋

### CODE REVIEW

File: src/pages/AddReservation.js [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
+
 import ReservationForm from '../components/ReservationForm';
+import doctorsDefault from '../assets/doctors-form-bg.jpg';
+
+const backgroundStyle = {
+  background: `url(${doctorsDefault}) no-repeat center center fixed`,
+  backgroundSize: 'cover',
+  backgroundColor: '#afd917',
+  opacity: '0.7',
+  backgroundBlendMode: 'multiply',
```
Comment (cvalencia1991) [inline src/pages/AddReservation.js:11]: Try to not use constants in the styles, this is a **bad practice**

File: src/App.css [hunk 2]
```diff
@@ -115,3 +119,12 @@
   width: 48px !important;
   right: 10px !important;
```
Comment (cvalencia1991) [inline src/App.css:120]: you should not use !important, this is a bad practice

File: src/components/DoctorBrief.js [hunk 3]
```diff
@@ -1,20 +1,46 @@
...
+            <a className="p-1 border-2 rounded-full border-gray-300" href="https://www.facebook.com/">
+              <TiSocialFacebook className="fill-gray-300" />
+            </a>
+            <a className="p-1 border-2 rounded-full border-gray-300" href="https://www.instagram.com/">
+              <TiSocialInstagram className="fill-gray-300" />
+            </a>
+          </div>
+        </div>
+      </div>
+      {/* Next div used to force carousel to work, do not remove */}
```
Comment (cvalencia1991) [inline src/components/DoctorBrief.js:38]: you should not put comments in your project, this is a bad practice

File: src/components/ReservationForm.js [hunk 4]
```diff
@@ -50,36 +50,34 @@ const ReservationForm = () => {
...
   return (
-    <div className="w-3/4 flex flex-col items-center">
+    <div className="w-3/4 flex flex-col py-4 items-center">
       { doctorId !== undefined && (
-        <div>
+        <div className="flex flex-col items-center pb-4 gap-2">
           <p>
-            Book an appointment with Dr.
+            Book an appointment with M.D.
             {' '}
```
Comment (cvalencia1991) [inline src/components/ReservationForm.js:58]: Try to delete the blank spaces, that generate the esilint, this is a bad practice

**Review by cvalencia1991 — CHANGES_REQUESTED** [review]
Hi @Camilovelag  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

# There is some issues in the test , i run all the test and appear me this, please try to fix it 

image

image



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

File: src/pages/AddReservation.js (hunk 1 above)
Comment (Camilovelag) [reply src/pages/AddReservation.js:11]: Done! 😄

File: src/components/ReservationForm.js (hunk 4 above)
Comment (Camilovelag) [reply src/components/ReservationForm.js:58]: Done! 😄

File: src/components/DoctorBrief.js (hunk 3 above)
Comment (Camilovelag) [reply src/components/DoctorBrief.js]: Done! 😄

File: src/App.css (hunk 2 above)
Comment (Camilovelag) [reply src/App.css:120]: According to @asdt560, this is needed to do responsive navPanel works, but it will be reviewed afterward.

**Review by cvalencia1991 — APPROVED** [review]

**Pull request merged**